model_compression_toolkit/core/pytorch/back2framework/pytorch_model_builder.py

```diff
@@ -188,14 +210,27 @@ def _add_modules(self):
         for n in self.node_sort:
             if isinstance(n, FunctionalNode):
                 # for functional layers
-                setattr(self, n.name, self.wrapper(n, n.type))
+                setattr(self, n.name,
+                        self.wrapper(n,
+                                     n.type,
+                                     include_activation_quantizers=not self.use_activation_holder_during_model_building)
```

Comment:
Not sure, just want to verify:
What would happen if we have an activation node (e.g., Relu) and we want to quantize its activation using a holder not a wrapper?
If I understand the flow of the code - it would be wrapped in this line, with "include_activation_quantizers=False", so basically unnecessary wrapping, and then a holder would be added after it (that's fine).
If that's the case then that is not correct, and we need to prevent it from being wrapped.

Reply:
After verifying it - it seems fine, no problem here